rados/watcher.go

```diff
@@ -345,7 +347,7 @@ func watchNotifyCb(_ unsafe.Pointer, notifyID C.uint64_t, id C.uint64_t,
 	watchersMtx.RUnlock()
 	if !ok {
 		// usually this should not happen, but who knows
-		// TODO: some log message (once we have logging)
+		log.Warnf("received notification for unknown watcher ID")
```

Comment:
If possible, I think it would be good to log the ID value that was not known.  Even if its not clear to "us" what the ID is, someone doing deeper debugging could possibly correlate the ID to something in other logs or in a dump of wire protocol data.

Reply:
right. done.